templates/base/packages/nextjs/package.json

```diff
@@ -39,7 +39,6 @@
     "@trivago/prettier-plugin-sort-imports": "~4.1.1",
     "@types/node": "~17.0.45",
     "@types/react": "~18.0.21",
```

Comment:
Changing this to `~18.2.24` fixes the issue

Reply:
Maybe having `^` instead of `~` or a fixed version for all the `@types` packages could solve this issue (and future issues too?)

Reply:
It seems pretty standard on all the sub-deps in `@types` packages.

Reply:
Yup 🙌 I think adding `^` seems fair solution, I tried fixing @types/react to `18.0.21` and it didn't work 

Also btw I think we should make a quick another patch PR cli  because `npx create-eth@latest` is broken now because of this